packages/core/app/styles/navi-core/components/navi-table-sort-icon.less

```diff
@@ -7,14 +7,20 @@
   display: flex;
   flex-flow: column;
   margin-left: 5px;
+  margin-top: -2px;
 
   &__caret {
     height: 0;
     color: @navi-lighter-gray;
     font-size: @font-size-base;
   }
 
-  &--asc .sort-icon__caret-up,
+  &--asc .sort-icon__caret-down {
+    color: @navi-gray;
+    transform: scaleY(-1);
```

Comment:
I'd prefer if we used an `if` in the template to decide which icon to use instead of applying a transform to flip the caret icon. I think it will be easier for someone in the future to know how it works and also the element class will not have `fa caret-down` in it when it is actually facing upwards.

Reply:
I don't mind this, but we should change the name of the class to just .sort-icon__caret--asc and .sort-icon__caret--desc

Reply:
The element will still have the font awesome class name in it though, so when you look at the dom, the ascending icon will still have `fa-caret-down` or whatever the icon name is on it even though it is pointing upwards.

Reply:
I agree that would be confusing, if you are just looking at the dom. Can we switch the icon dynamically?